setup.cfg

```
@@ -47,7 +47,8 @@ doc =
 test =
     katsdpsigproc[CUDA]
     async-solipsism
-    pytest
+    async-timeout
+    pytest>8
```

```suggestion
    pytest>=8
```
It's standard for lower bounds to be `>=`. What you've written means "pytest 8.x+, except for 8.0.0".

---

I think this was my attempt at ~coercing it to get a version newer than 8.0.x. In the end I manually set it to 8.2.2 in `requirements-dev.txt`.
* I'll update to `>=`